packages/vscode-extension/src/handlers.ts

```diff
@@ -1422,7 +1422,7 @@ export async function getFuncPathHandler(): Promise<string> {
     const vscodeDepsChecker = new VSCodeDepsChecker(vscodeLogger, vscodeTelemetry);
     const funcStatus = await vscodeDepsChecker.getDepsStatus(DepsType.FuncCoreTools);
     if (funcStatus?.details?.binFolders !== undefined) {
-      return `${path.delimiter}${funcStatus.details.binFolders.join(path.delimiter)}`;
+      return `${funcStatus.details.binFolders.join(path.delimiter)}${path.delimiter}`;
```

Comment:
Add `${path.delimiter}` to both front and end.

Reply:
Added.